internal/credentials/credentials.go

```diff
@@ -27,9 +32,9 @@ const (
 	// TypeImage represents credentials for an image repository.
 	TypeImage Type = "image"
 
-	// credentialTypeLabelKey is the key for a label used to identify the type
+	// CredentialTypeLabelKey is the key for a label used to identify the type
 	// of credentials stored in a Secret.
-	credentialTypeLabelKey = "kargo.akuity.io/cred-type" // nolint: gosec
+	CredentialTypeLabelKey = "kargo.akuity.io/cred-type" // nolint: gosec
```

Comment:
Wondering if this actually shouldn't be moved into the API package.

Reply:
I was initially keeping it close to the _one_ place it was used.

Now that it's called for in more than one place, I agree.

I _also_ have a need to move it in a PR I am working on.

Reply:
Moved it in 0606a09